README.md

```diff
@@ -4,79 +4,136 @@
 
 Inspired by Eric Meyers CSS Reset, Git New is a simple tool to enforce a consistent baseline Git experience for new projects.
 
-## Installation
+- [Git New](#git-new)
 
-Linux, Windows and OSX
+  - [For Project Developers](#for-project-developers)
+    - [Prerequisites](#prerequisites)
+    - [Setup and Configuration](#setup-and-configuration)
+  - [For Installers](#for-installers)
+    - [Prerequisites](#prerequisites-1)
+    - [Setup and Configuration](#setup-and-configuration-1)
+  - [Change log](#change-log)
+  - [Meta](#meta)
+  - [Contributing](#contributing)
+
+## For Developers
+
+Follow this section if you are developing with, installing or deploying this repository.
+
+### Prerequisites
+
+- A good code editor. We suggest [VSCode](https://code.visualstudio.com/), [Atom](https://atom.io/) or [SublimeText](https://www.sublimetext.com/).
```

Comment:
Although I like promoting good tools, is the IDE necessary for the README? The project can be run, edited, maintained, built, deployed, etc. all without the IDEs listed. Despite the incredible usefulness of the tools, I wouldn't say they are necessarily required the way Node or Git is. Thoughts?

Reply:
I dunno why I keep coming back to this. Yes, let's pull out the IDEs and editors.
```suggestion
- A good code editor.
```